src/components/organisms/SettingsManager/Settings.tsx

```diff
@@ -228,6 +264,34 @@ export const Settings = ({
           </>
         </Form>
       )}
+      <S.Div>
+        <S.Span>K8S Version</S.Span>
+        <div>
+          <Tooltip title="">
```

Comment:
missing tooltip - suggest "Select which kubernetes schema version to use for validation"